### ISSUE TITLE
FEW MORE Additions 

### ISSUE THREAD

**Issue by theRay1s** [adekmaulana/caligo#102]
Sir Add Language Translate Plugin Support

**Comment by tomyprs** [comment]
> Sir Add Language Translate Plugin Support

I don't think this modules really needed, because telegram official have built-in app, and very easiest than use userbot.
